Dockerfile

```diff
@@ -14,19 +14,16 @@ RUN  ./build.sh
 
 CMD ["./dist/hydrophone"]
```

Comment:
Useful ?

Reply:
If you want to run the "development" image, yes it is.